lib/Expo.php

```diff
@@ -148,12 +160,18 @@ private function prepareCurl()
     {
         $ch = $this->getCurl();
 
+        $headers = [
+                'accept: application/json',
+                'content-type: application/json',
+        ];
+
+        if ($this->accessToken) {
+            $headers []= sprintf('authorization: bearer %1$s', $this->accessToken);
```

Comment:
Just minor thing `$headers[] = sprintf('Authorization: Bearer %s', $this->accessToken);`